core/templates/dev/head/domain/exploration/WrittenTranslationsObjectFactory.js

```diff
@@ -84,9 +83,6 @@ oppia.factory('WrittenTranslationsObjectFactory', [
     WrittenTranslations.prototype.addWrittenTranslation = function(
```

Comment:
From the below change, it looks like this function is being used to both update and add new translations. If so, maybe a better name is updateWrittenTranslation